common/util.go

```diff
@@ -22,7 +22,6 @@ import (
 	fileservice "github.com/Azure/azure-sdk-for-go/sdk/storage/azfile/service"
 )
 
-var AzcopyJobPlanFolder string
```

Comment:
@adreed-msft should we maybe keep this global (instead of defined in the client) but put it in like the init file so its easy to discover?

Reply:
nevermind I thought about this some more and I think I like this variable existing in common. Will move this to init though for easy tracking